### ISSUE TITLE
Brotli adds "content-language: br"

### ISSUE THREAD

**Issue by nlemoine** [h5bp/server-configs-apache#248]
I recently started to serve brotli pre-compressed files. Works fine except that all files served have a `content-language: br` (brazilian) unwanted header. 

I stumbled upon this post: https://blog.desgrange.net/post/2017/04/10/pre-compression-with-gzip-and-brotli-in-apache.html

The mime module seems to blame. Could you fix this by adding a `RemoveLanguage .br` directive?

**Comment by LeoColomb** [comment]
Thanks for opening this issue @nlemoine!

> Works fine except that all files served have a `content-language: br` (brazilian)

Really interesting case!
Did you actually see this behavior yourself?
What was your version?

> Could you fix this by adding a `RemoveLanguage .br` directive?

Looks fair enough. Please open a PR! 🙂

**Comment by nlemoine** [comment]
> Did you actually see this behavior yourself?

Yes, I noticed it when setting up brotli pre compression but didn't pay attention. I stumbled upon this post for another reason and adding the fix. 

> What was your version?

Apache 2.4.34 (on a shared hosting)

While submitting the PR, I took a deeper look at this issue. This seems to be the [regular behaviour of mod_mime](https://httpd.apache.org/docs/2.4/en/mod/mod_mime.html#multipleext).
